.github/workflows/build.yml

```diff
@@ -106,13 +121,26 @@ jobs:
           # OSs not already tested above
           - os: ubuntu-22.04
             artifact_name: capa
-            asset_name: linux
+            asset_name: linux-amd64
+          - os: ubuntu-22.04-arm
+            artifact_name: capa
+            asset_name: linux-arm64
           - os: ubuntu-22.04
             artifact_name: capa
             asset_name: linux-py312
```

Comment:
```suggestion
            asset_name: linux-amd64-py312
```

Reply:
No longer relevant due to https://github.com/mandiant/capa/pull/2691#discussion_r2138740671